src/abstractdataframe/show.jl

```diff
@@ -30,7 +30,7 @@ end
 #'
 #' ourstrwidth("abc")
 #' ourstrwidth(10000)
-begin
+let
     local io = IOBuffer(Array(UInt8, 80), true, true)
```

Comment:
I think this is meant to be `let io = IOBuffer` on a single line. Not totally sure, but that's memory of how `let` blocks work. I don't think you need a `local` once you move it to the first line.